src/services/BookLoader.tsx

```diff
@@ -15,10 +14,8 @@ type Data = {
   shelves: ShelfDetail[]
 };
 
-const adapter = new JSONFile<Data>(LIBRARY_DB);
 const defeaultData = { books: [], shelves: [] };
-const libraryDb = new Low(adapter, defeaultData);
-await libraryDb.read();
+const libraryDb = await JSONFilePreset<Data>(LIBRARY_DB, defeaultData);
```

Comment:
Please correct spelling 'defeaultData' to 'defaultData'.

Reply:
I've fixed the typo